src/figuresClasses.ts

```diff
@@ -1,11 +1,95 @@
-export interface Figure {}
+// import { error } from 'console';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: `red` | `green` | `blue`;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
```

Comment:
The `shape` property should be declared as a class property with a type annotation, similar to `color`. It should be `shape: 'triangle' = 'triangle';` to properly initialize and type the property.